content/en/docs/tasks/observability/distributed-tracing/configurability/telemetry-api/index.md

```diff
@@ -0,0 +1,190 @@
+which allows label-based selection of workloads.
+
+It is not valid to have two different `Telemetry` resources select the same workload using `selector`. Likewise, it is not valid to have two
+distinct `Telemetry` resources in a namespace with no `selector` specified.
+
+### Scope, Inheritance, and Overrides
+
+Telemetry API resources inherit configuration from parent resources in the Istio configuration hierarchy:
+
+1.  root configuration namespace (example: `istio-system`)
+1.  local namespace (namespace-scoped resource with **no** workload `selector`)
+1.  workload (namespace-scoped resource with a workload `selector`)
+
+A Telemetry API resource in the root configuration namespace, typically `istio-system`, provides mesh-wide defaults for behavior.
+Do not specify any workload-specific selector in the root configuration namespace (it will be ignored).
+
+Namespace-specific overrides for the mesh-wide configuration can be achieved by applying a new `Telemetry` resource in the desired
+namespace (without a workload selector). Any `Tracing` fields specified in the namespace configuration will completely override
+the field from the parent configuration (in the root configuration namespace).
+
+Workload-specific overrides can be achieved by applying a new Telemetry resource in the desired namespace *with a workload selector*.
+Any `Tracing` fields specified in the namespace configuration will completely override the field from any parent configuration
+(root configuration or local namespace).
+
+## Using the Telemetry API for Tracing configuration
+
+### Configuring tracing providers
+
+Tracing providers are the backend collectors and processors that receive tracing spans
+and process them for storage and retrieval. Example providers include Zipkin, Jaeger, Lightstep,
+Datadog, and Apache SkyWalking.
+
+For Istio, tracing providers are configured for use within the mesh via `MeshConfig`. To configure new
+providers to use in tracing, edit the `MeshConfig` for your mesh via:
+
+{{< text bash >}}
+$ kubectl -n istio-system edit configmap istio
+{{< /text >}}
+
+The full set of configuration options is described in the [reference docs for MeshConfig](/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig).
+Typical configuration includes service address and port for the provider, as well as establishing
+a limit on max tag length supported by the provider.
+
+Each configured provider *must* be uniquely named. That name will be used to refer to the provider in the
+Telemetry API.
+
+An example set of provider configuration in `MeshConfig` is:
+
+{{< text yaml >}}
+data:
+  mesh: |-
+      extensionProviders: # The following content defines two example tracing providers.
+      - name: "zippy-kin"
+        zipkin:
+          service: "zipkin.istio-system.svc.cluster.local"
+          port: 9411
+          maxTagLength: 56
+      - name: "stacky-driver"
```

Comment:
Why the cutesy names?

Reply:
because life is long and full of pain and suffering, doubly so when reading through technical documentation for a service mesh product.

in the end, i just didn't want to give people the impression that `provider` names needed to follow any sort of scheme. they can be anything.

i've updated to use different names throughout.